db/migrate/20170118232430_create_join_table_authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableAuthenticationConfigurationScriptBase < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_configuration_script_bases do |t|
```

Comment:
For record purpose: We'll stick with `HMT` and use the naming convention of `HMT` which is `singular-plural`. 
For reference: 
* http://stackoverflow.com/questions/33045170/join-table-for-has-many-through-in-rails 
* https://gitter.im/ManageIQ/manageiq?at=562ed95c0791816c7a089e26